Gems/Atom/RPI/Code/Source/RPI.Public/MeshDrawPacket.cpp

```diff
@@ -422,10 +422,13 @@ namespace AZ
                         drawSrg->SetConstant(index, uvStreamTangentBitmask.GetFullTangentBitmask());
                     }
 
-                    // We need to call Reset for each ShaderItem, because not all shader have
-                    // the DrawSrg::m_modelLodMeshIndex constant defined in them.
-                    m_drawSrgModelLodMeshIndex.Reset();
-                    drawSrg->SetConstant(m_drawSrgModelLodMeshIndex, aznumeric_cast<uint32_t>(m_modelLodMeshIndex));
+                    AZ::Name drawSrgModelLodMeshIndex = AZ::Name(DrawSrgModelLodMeshIndex);
```

Comment:
Is it possible to cache the AZ:Name + index validity once instead of calculating it every time this function is called?

Reply:
There's no need, as this code only runs during initialization or material changes. It doesn't run every frame.

It's doing exactly as the code just above related with:
```
AZ::Name shaderUvStreamTangentBitmask = AZ::Name(UvStreamTangentBitmask::SrgName);
```